projects/cds/src/merchandising/cms-components/merchandising-carousel/merchandising-carousel.component.service.ts

```diff
@@ -44,22 +47,14 @@ export class MerchandisingCarouselComponentService {
   }
 
   private getCarouselMetadata(
-    merchandisingProducts: MerchandisingProducts,
+    strategyProducts: StrategyProducts,
```

Comment:
since we changed it from MerchandisingProducts to StrategyProducts, is there a need to keep  MerchandisingProduct? Since we will use the product id from the response given from the strategy endpoint and map that id to the occ product, going back with what I said, is there a need for MerchandisingProduct?

Reply:
I think I'm wrong but just another clarification on it would be nice